Explorer/Assets/DCL/Multiplayer/Connections/Archipelago/Rooms/Fixed/FixedConnectiveRoom.cs

```diff
@@ -0,0 +1,86 @@
+using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using DCL.Multiplayer.Connections.Archipelago.AdapterAddress.Current;
+using DCL.Multiplayer.Connections.Rooms.Connective;
+using DCL.WebRequests;
+using LiveKit.Rooms;
+using System;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Connections.Archipelago.Rooms.Fixed
+{
+    public class FixedConnectiveRoom : IConnectiveRoom
+    {
+        private readonly IWebRequestController webRequests;
+        private readonly IConnectiveRoom connectiveRoom;
```

Comment:
I have a feeling we never use `ConnectiveRoom` as `IConnectiveRoom`  that can be used from outside by consumers. With every new implementation of the same interface I will be challenging with more questions. it feels like the internal core should not be under the interface as it's hardwired into the implementation.
The same concern relates to `ArchipelagoIslandRoom`

Reply:
Consumer is RoomHub, it uses them

Reply:
Considering composed object as one unit it won't be a problem of amount of classes, each class is small, maintainable and easy to refactor, to see how it works together could be viewed at the composition root

Reply:
If `ConnectiveRoom` is a core:
- Can it be used as a core without interfaces?
- Can it have the name with `Core`
- Only referenced from `ArchipelagoRoom`, `GateKeeperSceneRoom`, `FixedConnectiveRoom` from where it should be used by design. Then it will be clear how the core is connected with additional functionality as the core can't be used in practice without the corresponding higher level `IConnectiveRoom`